draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,27 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers {#subscription-id}
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a Track, it associates a Request ID with the request.  The
+Request ID (RSID) can be used as a Subscription ID by shifting it left by one,
+eg: `Request Subscription ID = Request ID << 1`.
```

Comment:
I am still not clear, what RequestID is used to generate RSID . Also when should one use which (ReqID vs RSID).

Reply:
SUBSCRIBE contains a Request ID, as do (almost) all Control Messages, since we landed 905.  That said, you will never send Request ID *directly* on the wire after SUBSCRIBE/SUBSCRIBE_OK/SUBSCRIBE_ERROR as this PR is currently written - all other operations use Subscription ID.  You can scan the Message formats below to see if they use Request ID or Subscription ID.